README.md

```diff
@@ -16,6 +16,12 @@ yarn global add ember-no-implicit-this-codemod
 ember-no-implicit-this-codemod <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
 ```
 
+The codemod accepts the following options:
+
+| Option | Value | Default | Details |
+| --- | --- | ---| --- |
+| --conservative-mode | boolean | false | The codemod won't prepend `this.` to `PathExpressions` names which don't appear in a component's telemetry data |
```

Comment:
I'm definitely not tied to `conservative-mode` so please let me know if you think there's a better name. Naming is hard 😅

Reply:
maybe `--dont-assume-this-if-property-not-found`?

Reply:
That seems a bit verbose and requires _lots_ of typing 😬

How about `--dont-assume-this` instead? Hopefully the description of the option in the table is descriptive enough that we don't need to fully describe it in the option name? I can update the description if not.

On the other hand, if you feel strongly about using `--dont-assume-this-if-property-not-found` I'll make the change.